MapboxVisionAR/Render/ARScene/ConcreteNodes/ARCameraNode/ARCameraNode.swift

```diff
@@ -0,0 +1,70 @@
+import simd
+
+class ARCameraNode: ARNode {
+    // MARK: - Properties
+
+    /// Type of node. Always returns `rootNode`.
```

Comment:
It won't return `rootNode`, it will return `cameraNode`, you even have a test for it

Reply:
You're right, fixed.